src/main/java/frc/robot/subsystems/swerve/swervestatehelpers/AimAssistUtils.java

```diff
@@ -0,0 +1,41 @@
+package frc.robot.subsystems.swerve.swervestatehelpers;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.kinematics.ChassisSpeeds;
+import frc.robot.subsystems.swerve.SwerveConstants;
+import frc.robot.subsystems.swerve.SwerveMath;
+
+import java.util.function.Supplier;
+
+import static frc.robot.subsystems.swerve.SwerveMath.getDriveMagnitude;
+
+public class AimAssistUtils {
+
+    public static ChassisSpeeds getRotationAssistedSpeeds(
+            ChassisSpeeds wantedSpeeds,
+            Rotation2d robotRotation,
```

Comment:
```suggestion
            Rotation2d currentRotation,
```
you can do swerve.getHeading too just sayin, your choice do what you prefer